src/components/organism/sections/home/completionTools/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+import { Section } from "@/components/atoms/section";
+
+
+export const CompletionTools: React.FC = () => {
```

Comment:
**Change**: use export default function CompletionTools(). instead of this one

Reply:
Done